### ISSUE TITLE
Help to find errors

### ISSUE THREAD

**Issue by EdgarEspitia** [ec-jrc/lisflood-code#107]
Hi lisflood community,

I am trying to run lisflood, but I got some errors. This is the output:

```
LisfloodPy  4.0.0   09/06/2022

Water balance and flood simulation model for large catchments

(C) Institute for Environment and Sustainability
    Joint Research Centre of the European Commission
    TP122, I-21020 Ispra (Va), Italy


Start Step - End Step:  1  -  11688
Start Date - End Date:  1990-01-02 06:00:00  -  2022-01-01 06:00:00
Traceback (most recent call last):
  File "/mysystem/lisflood-code/src/lisf1.py", line 34, in <module>
    sys.exit(main())
  File "/mysystem/lisflood-code/src/lisflood/main.py", line 222, in main
    lisfloodexe(lissettings)
  File "/mysystem/lisflood-code/src/lisflood/main.py", line 115, in lisfloodexe
    Lisflood = LisfloodModel()
  File "/mysystem/lisflood-code/src/lisflood/Lisflood_initial.py", line 132, in __init__
    self.readmeteo_module = readmeteo(self)
  File "/mysystem/lisflood-code/src/lisflood/hydrological_modules/readmeteo.py", line 40, in __init__
    self.forcings[data] = xarray_reader(data)
  File "/mysystem/lisflood-code/src/lisflood/global_modules/netcdf.py", line 276, in xarray_reader
    data = XarrayChunked(data_path, time_chunk, dates, indexer, climatology)
  File "/mysystem/lisflood-code/src/lisflood/global_modules/netcdf.py", line 168, in __init__
    ds = xr.open_mfdataset(data_path, engine='netcdf4', chunks={'time': time_chunk}, combine='by_coords')
  File "/mysystem/conda_venvs/lisflood/lib/python3.7/site-packages/xarray/backends/api.py", line 873, in open_mfdataset
    raise OSError("no files to open")
OSError: no files to open
```
I prepared the setting file following the step-by-step user guide (https://ec-jrc.github.io/lisflood-code/),  I have all the inputs maps, the statics, and meteorological forcings, but I don't know how to start to trace where are the errors. I would be grateful if you can suggest or give some guidelines to continue.

Thanks

**Comment by StefaniaGrimaldi** [comment]
Hi @EdgarEspitia, 

thank you for sharing the error message.

The issue seems to be related to the reading module of the meteorological forcings:

  _File "/mysystem/lisflood-code/src/lisflood/hydrological_modules/readmeteo.py", line 40, in __init__
    self.forcings[data] = xarray_reader(data)_

As a first step, I would double check the path and the file name of the meteorological forcings in the settings .xml file.
For instance, for the total precipitation, I would check the settings in three different rows of the settings .xml:
<textvar name="PathMeteo" value= ''....'' >
<textvar name="PrecipitationMaps" value="$(PathMeteo)/$(PrefixPrecipitation)">
<textvar name="PrefixPrecipitation" value=".....">
I would repeat the check for all the meteorological forcings: total precipitation, average temperature, reference (evapo)transpiration.

Please feel free to share your settings .xml file, we could have a look at it together.

If all the paths and the file names are correct, we will have to investigate further.

Stefania

**Comment by EdgarEspitia** [comment]
Hi @StefaniaGrimaldi,

The issue was related to the path where my settings.xml was located, as I used the XML variable $(PathRoot) in the wrong folder. I solved the problem just by relocating my settings.xml.

After solving the problem with the meteorological forcings, It appeared more problems related to the length of the inputs files names. Finally, I was able to solve them, so I am running LISFLOOD. 

Thank you very much!
